docs/maya/tools.md

```diff
@@ -28,15 +28,10 @@ To open the tool:
 
 If the selection provided is valid, meaning the selected mesh has one of the AdonisFX deformers listed before, then the paint context will get configured and the user can paint. The map to be painted is the one associated to the selected attribute in the enumerator exposed at the top of the UI.
 
-The painted weights are visualised in the viewport thanks to the **AdnWeightsDisplayNode**. This node is in charge of setting the vertex colours from the selected attribute in the UI on the mesh. This node is created and removed dynamically by the Paint Tool on entering and exiting from the context. Because of that, it is strongly recommended to keep the AdonisFX Paint Tool open while painting.
-
-<figure markdown>
-  ![AdnWeightsDisplayNode in the Node Editor](images/paint_tool_weights_display_node.png) 
-  <figcaption><b>Figure 2</b>: AdnWeightsDisplayNode created by the AdonisFX Paint Tool. </figcaption>
-</figure>
-
 > [!NOTE]
-> - If the painted weights are not displayed, it is probably because the node got removed. Please, select the transform node of an AdonisFX paintable deformer and click *Refresh From Selection* or restart the AdonisFX Paint Tool.
+> - With the optimizations introduced to the AdonisFX Paint Tool in version 1.4.0, the AdnWeightsDisplayNode is no longer needed and has been deprecated in this release.
```

Comment:
Maybe is not necessary to expose here that AdnWeightsDisplayNode has been deprecated thanks to the optimizations introduced to the AdonisFX Paint Tool. Let me know what you think.

Reply:
I think it is ok! Just a small suggestion
```suggestion
> - With the optimizations introduced to the AdonisFX Paint Tool in version 1.4.0, the AdnWeightsDisplayNode is deprecated and no longer needed.
```

Reply:
Done!